waypoints/Alps-mountains.cup

```diff
@@ -0,0 +1,3884 @@
+"Hohe Salve",HoheSlve,AT,4727.800N,01212.466E,1827.0m,7,,,,"Gipfelhaus"
+"Hohe Schrott",HoheSott,AT,4743.457N,01342.056E,1839.0m,7,,,,"Gipfel"
+"Hohe Viller-Spitze",HoheVtze,AT,4706.383N,01109.833E,3092.0m,7,,,,"Gipfel"
+"Hohe Wand",HoheWand,AT,4700.866N,01137.583E,3289.0m,7,,,,"Gipfel"
+"Hohe Wand (Plackless)",HoheWPla,AT,4748.799N,01600.183E,1132.0m,7,,,,"Gipfel"
+"Hohe Wand gr. Kanzel",HoheWkKa,AT,4748.649N,01600.716E,1052.0m,7,,,,"Gipfel"
+"Hohe Wand kl. Kanzel",HoheWKan,AT,4749.766N,01600.966E,1065.0m,7,,,,"Gipfel"
+"Hohe Wand v. d. Wand",HoheWand,AT,4751.133N,01604.150E,915.0m,7,,,,"Gipfel"
+"Hohe Warte",HoheWrte,AT,4636.483N,01253.083E,2780.0m,7,,,,"Gipfel"
+"Hohe Warte",HoheWrte,AT,4703.766N,01135.283E,2687.0m,7,,,,"Gipfel"
+"Hohe Wilde",HoheWlde,AT,4646.066N,01101.200E,3482.0m,7,,,,"Gipfel"
+"Hohenwart",Hohenart,AT,4719.783N,01413.983E,2361.0m,7,,,,"Gipfel"
+"Hoher Bal",HoherBal,AT,4709.516N,01233.500E,2671.0m,7,,,,"Gipfel"
+"Hoher Burgstall",Hoherall,AT,4707.983N,01116.683E,2613.0m,7,,,,"Gipfel"
+"Hoher Dachstein",Hoherein,AT,4728.483N,01336.433E,2995.0m,7,,,,"Gipfel"
+"Hoher Frassen",Hoherssen,AT,4711.553N,00950.056E,1979.1m,7,,,,"Gipfel"
+"Hoher Freschen",Hoherhen,AT,4718.483N,00946.816E,2005.9m,7,,,,"Gipfel"
+"Hoher Goell",Hoheroell,AT,4735.186N,01304.439E,2523.0m,7,,,,"Gipfel"
+"Hoher Herd",Hohererd,AT,4711.949N,01226.966E,2824.0m,7,,,,"Gipfel"
+"Hoher Ifen",Hoherfen,AT,4721.250N,01006.166E,2230.0m,7,,,,"Gipfel"
+"Hoher Lorenzen-Berg",Hohererg,AT,4658.383N,01126.000E,2313.0m,7,,,,"Gipfel"
+"Hoher Riffler",Hoherler,AT,4704.983N,01142.333E,3231.0m,7,,,,"Gipfel"
+"Hoher Riffler",Hoherler,AT,4707.033N,01022.383E,3168.0m,7,,,,"Gipfel"
+"Hoher Sarstein",Hoherein,AT,4736.071N,01341.932E,1975.0m,7,,,,"Gipfel"
+"Hoher Seeblas Kogl",Hoherogl,AT,4705.700N,01104.283E,3235.0m,7,,,,"Gipfel"
+"Hoher Sonnblick",Hoherick,AT,4703.216N,01257.350E,3105.0m,7,,,,"Gipfel"
+"Hoher Student",Hoherstu,AT,4744.383N,01523.450E,1539.0m,7,,,,"Gipfel"
+"Hoher Tenn",Hoherenn,AT,4710.750N,01245.500E,3368.0m,7,,,,"Gipfel"
+"Hoher Trieb",Hoherieb,AT,4635.866N,01303.683E,2199.0m,7,,,,"Gipfel"
+"Hoher Wasserfall",Hoherall,AT,4709.833N,01058.200E,3002.0m,7,,,,"Gipfel"
+"Hoher Zinken",HoherZin,AT,4716.350N,01420.716E,2222.0m,7,,,,"Gipfel"
+"Hohes Kreuz",Hoheseuz,AT,4700.550N,01213.650E,3156.0m,7,,,,"Gipfel"
+"Hohes Licht",Hohescht,AT,4716.683N,01016.466E,2652.0m,7,,,,"Gipfel"
+"Hollbruck-Spitze",Hollb-Sp,AT,4641.950N,01225.550E,2581.0m,7,,,,"Gipfel"
+"Honigkogel",Honiggel,AT,4718.749N,01251.300E,1857.0m,7,,,,"Gipfel"
+"Ht Bbrunnen-Kogl",HtBbrogl,AT,4704.416N,01106.116E,3325.0m,7,,,,"Gipfel"
+"Ht. Daun-Kogl",Ht.Daogl,AT,4700.316N,01105.633E,3225.0m,7,,,,"Gipfel"
+"Ht. Schafbichl",Ht.Scchl,AT,4707.999N,01237.550E,2250.0m,7,,,,"Gipfel"
+"Ht. Seelenkogl",Ht.Seogl,AT,4648.166N,01102.533E,3470.0m,7,,,,"Gipfel"
+"Ht. Sonnenwend-J.",Ht.So-J.,AT,4735.900N,01157.100E,1986.0m,7,,,,"Gipfel"
+"Ht. Stangen-S",Ht.Stn-S,AT,4703.366N,01158.516E,3227.0m,7,,,,"Gipfel"
```

Comment:
Abbreviation consistency

Reply:
These are Huts and not mountain tops. I removed them.